stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -403,7 +400,12 @@ function _dense2sparsevec(s::AbstractArray{Tv}, initcap::Ti) where {Tv,Ti}
         resize!(nzind, c)
         resize!(nzval, c)
     end
-    SparseVector(n, nzind, nzval)
+    nzind, nzval
```

Comment:
I like explicit returns, especially when returning tuples.

```suggestion
    return (nzind, nzval)
```